static/js/simple_register_lookup_core.js

```diff
@@ -337,6 +343,20 @@ function RegistersTableRequester()
             CURRENT_APP_URL,
             data,
             function(data) {
+                for (i = 0; i < data.length; i++) {
+                  tagsToReplace = data[i][4].replace(" ","").split(",");
+                  tagsData = ""
+                  for(t = 0; t < tagsToReplace.length; t++) { 
+                     if(typeof TAG_MAPPINGS[tagsToReplace[t]] !== "undefined"){
+                        tagsData += '<a href="https://labjack.com'+ TAG_MAPPINGS[tagsToReplace[t]] +'" target="_blank">' + tagsToReplace[t]+ '</a>, '
+                        console.log(tagsToReplace[t]);
+                        console.log(TAG_MAPPINGS[tagsToReplace[t]]);
+                        }else{
+                          tagsData += tagsToReplace[t] + ", ";
+                        }
+                  }
+                  data[i][4] = tagsData.substring(0, tagsData.length - 2);
+               }
```

Comment:
For this section, please refactor your new code into a new function and stick with indentation of 4 spaces.